email/email_service.bal

```diff
@@ -60,43 +60,38 @@ service "EmailService" on new grpc:Listener(9097) {
         gmail:MessageRequest messageRequest = {
             recipient: request.email,
             subject: "Order Confirmation",
-            messageBody: (check self.getConfirmationHtml(request.'order)).toString(),
+            messageBody: (check getConfirmationHtml(request.'order)).toString(),
             contentType: gmail:TEXT_HTML
         };
-
         gmail:Message|error sendMessageResponse = self.gmailClient->sendMessage(messageRequest);
 
         if sendMessageResponse is gmail:Message {
             log:printInfo(string `Sent Message ID: ${sendMessageResponse.id}`);
             log:printInfo(string `Sent Thread ID: ${sendMessageResponse.threadId}`);
             return {};
         }
-        log:printError("Error sending confirmation mail ", 'error = sendMessageResponse);
-
+        log:printError("Error sending confirmation mail ", sendMessageResponse);
         return sendMessageResponse;
     }
+}
 
-    function getConfirmationHtml(stub:OrderResult result) returns xml|error {
-        string fontUrl =
+function getConfirmationHtml(stub:OrderResult result) returns xml|error {
+    string fontUrl =
                     "https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap";
```

Comment:
Should this be a constant?

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b